samples/ComputerVision/DomainSpecificContent/Program.cs

```diff
@@ -41,27 +41,27 @@ public static async Task RunAsync(string endpoint, string key)
             string remoteImageUrl = "https://github.com/Azure-Samples/cognitive-services-sample-data-files/raw/master/ComputerVision/Images/landmark.jpg";
 
             Console.WriteLine("Domain Specific Content being analyzed ...");
-            await DomainSpecificContentFromUrlAsync(computerVision, remoteImageUrl);
-            await DomainSpecificContentFromStreamAsync(computerVision, localImagePath);
+            await DomainSpecificContentFromUrlAsync(computerVision, remoteImageUrl, "landmarks");
+            await DomainSpecificContentFromStreamAsync(computerVision, localImagePath, "celebrities");
         }
 
         // Analyze a remote image
-        private static async Task DomainSpecificContentFromUrlAsync(ComputerVisionClient computerVision, string imageUrl)
+        private static async Task DomainSpecificContentFromUrlAsync(ComputerVisionClient computerVision, string imageUrl, string specificDomain)
```

Comment:
RecognizeDomainSpecificContentFromUrlAsync would be a better name (method names work better when they start with a verb). Please also change the folder/project name to reflect that.